catalog/app/containers/Auth/saga.js

```diff
@@ -150,6 +150,23 @@ function* signIn(credentials) {
   }
 }
 
+function* setBrowseCookie(tokens) {
```

Comment:
i think maybe we better should put this into `storeTokens()` (and maybe even unset the cookie in `forgetTokens`?) in `Provider.js`

Reply:
Hmm. Do you think it's a good idea to make API calls from `forgetTokens`?

As for `forgetTokens`: we could unset it... though as long as the `signOut` call succeeds, the cookie will become invalid anyway.

Reply:
> Hmm. Do you think it's a good idea to make API calls from storeTokens?

maybe it's not perfect, but i think it's the right place, since setting cookies is conceptually similar to writing to the local storage, even though the actual implementation (making an http request) is different

> As for forgetTokens: we could unset it... though as long as the signOut call succeeds, the cookie will become invalid anyway.

well, i guess that's ok

Reply:
your approach would work just fine, nevermind